src/uwtools/config/support.py

```diff
@@ -1,8 +1,10 @@
 from __future__ import annotations
 
+from collections import OrderedDict
 from importlib import import_module
 from typing import Dict, Type, Union
 
+import f90nml  # type: ignore
```

Comment:
Here, too, maybe `from f90nml import Namelist` unless there are other `f90nml` items needed in this module.